articles/post-wsuwp_uc_entity.php

```diff
@@ -0,0 +1,29 @@
+<article id="post-<?php the_ID(); ?>" <?php post_class(); ?>>
+
+	<header class="article-header">
+		<hgroup>
+			<?php if ( is_single() ) : ?>
+				<h1 class="article-title"><?php the_title(); ?></h1>			<?php else : ?>
```

Comment:
This `<?php else: ?>` isn't necessary if you don't need to output anything for archive views. And if that is indeed the case, you might consider moving the `is_single()` check outside of the `<header>` element (example below) so we're not outputting empty markup.
```php
<?php if ( is_single() ) : ?>
<header class="article-header">
...
</header>
<?php endif; ?>```